### DISCUSSION TITLE
OpenMMLab 中文社区答疑流程指南

### DISCUSSION THREAD

**Discussion by vansin** [open-mmlab/mmengine#552] [General]
# 如何在 OpenMMLab 社区获得帮助？ 

为了更高效的帮助社区同学解决问题，建议社区小伙伴在OpenMMLab GitHub 的 Issues 或 Discussions 详细描述问题及其复现路径,并把相关Issues或者Discussions链接发在群中~

**Comment by Gaofan666** [comment]
你好,mmdet.apis我这里显示不出来，其他的文档都有

**Comment by Gaofan666** [comment]
运行B4，准备训练部分代码的时候报错
![9e531acbcbaaaaf3965406f6cb98d22](https://user-images.githubusercontent.com/60742877/198758721-70720d87-65c0-4559-9782-9605da140a94.jpg)

**Comment by surelon** [comment]
老师您好！请问修改mmsegmentation里网络模型的结构有相关教程吗？搜了官方b站和知乎的资料没发现，mmcv很集成，我该怎么入手呢？比如修改一下损失函数，添加注意力机制模块啥的。

**Reply by vansin** [reply]
看看里面有没有想要的内容哇~
https://github.com/open-mmlab/OpenMMLabCourse

**Reply by surelon** [reply]
谢谢！正在看

**Comment by surelon** [comment]
请问现在mmsegmentation有EfficientNet的主干网络吗？我从configs里没有找到

**Reply by surelon** [reply]
路过的大佬请留步，还有一个问题请教，在mmsegmentation的loss库中（cross_entropy_loss，dice_loss，focal_loss，lovasz_loss，tversky_loss），哪个更适合用于解决数据中的类不平衡问题呢？

**Comment by AndrewJSong** [comment]
现在的文档跟代码版本不统一。比如文档中：runner = Runner.build_from_cfg(config)
实际上现在mmengine里面Runner已经没有build_from_cfg这个方法了

**Comment by NatsunoKiode** [comment]
![image](https://user-images.githubusercontent.com/118642974/202878674-e986b73e-4b85-487e-8e87-1ca4c51cc3ce.png)
ModuleNotFoundError: No module named 'tools.data_converter'  是因为少配置了部分库还是其他原因导致的 网上检索下来没用应对的方法

**Comment by Marc565** [comment]
![image](https://user-images.githubusercontent.com/123915890/215375910-b0469a77-7f8b-4949-bda8-29bb40c6ca84.png)
求大佬指路

**Comment by Zaughter** [comment]
目前mmDeploy是否支持DAB-DETR?
